src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -375,6 +375,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('autoReveal', "Controls whether the explorer should automatically reveal and select files when opening them."),
 			'default': true
 		},
+		'explorer.autoRevealNoScroll': {
```

Comment:
`explorer.autoRevealScroll` is a better name

Reply:
What about `explorer.autoScrollOnReveal` ?

As an end user that tells me exactly what's happening without even having to read the description.

Reply:
`autoScrollOnReveal` also highlights the dependency on `autoReveal` which you referenced here @phuein https://github.com/microsoft/vscode/pull/96890#discussion_r420491861